weaknesses/MASVS-PRIVACY/MASWE-0117.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Inadequate Permission Management
+id: MASWE-0117
+alias: inadequate-permission-management
+platform: ["android", "ios"]
+profiles: ["P"]
+mappings:
+  masvs-v1: []
+  masvs-v2: [MASVS-PRIVACY-1]
+  cwe: [250]
+refs:
+- https://developer.apple.com/design/human-interface-guidelines/privacy#Requesting-permission
+- https://developer.apple.com/documentation/uikit/protecting_the_user_s_privacy/requesting_access_to_protected_resources
+- https://developer.android.com/training/permissions/requesting#explain
+- https://support.google.com/googleplay/android-developer/answer/9888170?hl=en
+- https://developer.android.com/privacy-and-security/minimize-permission-requests
+- https://developer.android.com/training/permissions/requesting
+- https://developer.android.com/training/permissions/requesting#remove-access
+- https://developer.android.com/training/permissions/usage-notes
+- https://arxiv.org/pdf/1905.02713
+- https://arxiv.org/pdf/2203.10583
+- 
+
+status: new
+---
+
+## Overview
+
+Inadequate permission management poses significant risks to user privacy and security in mobile apps. Permissions control access to sensitive device features, such as the camera, microphone, location, and storage, which are essential for app functionality. However, improper management, such as requesting excessive or unnecessary permissions, can lead to privacy violations, unauthorized data access, and erosion of user trust. For example, some apps retain access to sensitive resources long after they are no longer needed (even when platforms like Android 13 offer mechanisms to revoke unused permissions). Developers face the challenge of balancing functionality with privacy, as revoking permissions can disrupt app usability, forcing users to choose between privacy and functionality.
+
+Some apps, especially those that come pre-installed on devices, are often granted excessive privileges without needing explicit consent. Also, some regular apps may request permissions that are not necessary for their core functionality or request broader access than needed. For example, granting access to the camera may also grant access to the photo gallery, reducing user control. Privacy-friendly alternatives (e.g., coarse location or image picker) are often ignored.
```

Comment:
```suggestion
Some apps, especially those that come pre-installed on devices, are often granted excessive privileges without needing explicit consent.
```

This is redundant to what is mentioned in the modes of introduction. We could remove it from here.